### PR TITLE
Allow telemetry devices use jvm.options instead of ES_JAVA_OPTS

### PR DESCRIPTION
**Pull request by ebadyano**

### CODE REVIEW

**Comment by ebadyano** [comment]
Unblocks https://github.com/elastic/rally/pull/711

File: cars/v1/vanilla/templates/config/jvm.options [hunk 1]
```diff
@@ -112,3 +112,10 @@
 # due to internationalization enhancements in JDK 9 Elasticsearch need to set the provider to COMPAT otherwise
 # time/date parsing will break in an incompatible way for some date patterns and locals
 9-:-Djava.locale.providers=COMPAT
+
+{# cluster settings provided by the user via the command line #}
```
Comment (danielmitterdorfer) [inline cars/v1/vanilla/templates/config/jvm.options]: Nit: The comment is not applicable here. I'd probably just get rid of it.

**Comment by danielmitterdorfer** [comment]
We should backport this as far as `jvm.options` are supported (Elasticsearch 5.0.0).

**Comment by ebadyano** [comment]
@danielmitterdorfer Thank you for the review!

**Pull request merged**